### PR TITLE
KFSPTS-17856 upgrade to the 3/19/2020 version of financials

### PR DESCRIPTION
**Pull request by db483**
There are PRs for this in cu-kfs, nonprod-sql and cu-kfs-upgrades. In addition, these changes are dependent on the KFSPTS-18970 upgrade changes for Cynergy.

Documentation on how to run the upgraded Cynergy and KFS locally: https://confluence.cornell.edu/x/mpL-Fg

### CODE REVIEW

**Comment by cah292** [comment]
@db483 Please let me know if you think another developer should also review this, given that you helped with some of the initial coding.

File: src/test/resources/edu/cornell/kfs/krad/service/impl/AccountCustomAddressTest.xml [hunk 1]
```diff
@@ -937,114 +937,6 @@
       <objectId>A53DCA7C3185467EE04054802FC207C1</objectId>
       <newCollectionRecord>false</newCollectionRecord>
     </organization>
-    <accountFiscalOfficerUser class="org.kuali.rice.kim.impl.identity.PersonImpl">
```
[the remark this answers is not in the document]
Comment (cah292) [reply src/test/resources/edu/cornell/kfs/krad/service/impl/AccountCustomAddressTest.xml:940]: Our current version of the upgrade rules was just modifying the PersonImpl objects to still be creatable from the XML. However, given the potential complexities of making these objects compatible with the KIM-to-KFS changes, I have updated our upgrade rules to just remove the PersonImpl objects, thus forcing the documents to just reload the PersonImpl data accordingly. (The base financials XML upgrade rules also appear to be set up to just remove the PersonImpl objects, at least as of the 2020-03-19 patch.)

File: src/test/resources/edu/cornell/kfs/krad/service/impl/VendorTest.xml [hunk 2]
```diff
@@ -793,35 +764,6 @@
       <newCollectionRecord>false</newCollectionRecord>
 
       <vendorW9SignedDate>2014-10-01</vendorW9SignedDate></vendorHeader>
-    <vendorRestrictedPerson class="org.kuali.rice.kim.impl.identity.PersonImpl">
```
[the remark this answers is not in the document]
Comment (cah292) [reply src/test/resources/edu/cornell/kfs/krad/service/impl/VendorTest.xml:796]: See my notes on the related case in AccountCustomAddressTest.xml

File: src/test/resources/edu/cornell/kfs/krad/service/impl/AccountTest.xml [hunk 3]
```diff
@@ -482,135 +482,6 @@
 
 
     <contractsAndGrantsAccountResponsibilityId>6</contractsAndGrantsAccountResponsibilityId>
-    <accountFiscalOfficerUser class="org.kuali.rice.kim.impl.identity.PersonImpl">
```
[the remark this answers is not in the document]
Comment (cah292) [reply src/test/resources/edu/cornell/kfs/krad/service/impl/AccountTest.xml:485]: See my notes on the related case in AccountCustomAddressTest.xml

File: src/test/resources/edu/cornell/kfs/krad/service/impl/AccountCustomAddressTest.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (db483) [reply src/test/resources/edu/cornell/kfs/krad/service/impl/AccountCustomAddressTest.xml:940]: Thank you for explaining this.

**Comment by cah292** [comment]
I'm closing and recreating this PR, so that it will be in a state where Daniela can approve and merge it.

**Pull request closed without merging**